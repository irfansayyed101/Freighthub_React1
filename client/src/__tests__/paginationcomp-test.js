import React from 'react';
import { shallow } from 'enzyme';
import PaginationComp from '../components/PaginationComp';

describe('<PaginationComp /> with props', () => {
    const initialFormState = { total: 24,
        per_page: 10,
        current_page: 1,
        limit:10
      };
      const container = shallow(<PaginationComp pagination={initialFormState}/>);
      it('should match the snapshot', () => {
        expect(container.html()).toMatchSnapshot();
      });
});