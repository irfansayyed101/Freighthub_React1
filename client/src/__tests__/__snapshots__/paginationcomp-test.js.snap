// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PaginationComp /> with props should match the snapshot 1`] = `"<div><div class=\\"pagination\\"><span class=\\"active\\">1</span><span class=\\"\\">2</span><span class=\\"\\">3</span></div></div>"`;
